app/controllers/open_search_controller.rb

```diff
@@ -0,0 +1,8 @@
+class OpenSearchController < ApplicationController
```

Comment:
For my own education, what is this for?

Reply:
We currently have `search.xml` in our assets, and this moves it to a controller for the purpose of this PR, but in general  it's a file to be compatible with this: https://developer.mozilla.org/en-US/docs/Web/OpenSearch ...which allows users to search through a site within their browser. It's not the most important thing in the world but it's a protocol we support.

<img width="468" alt="Screen Shot 2020-09-16 at 11 22 19 AM" src="https://user-images.githubusercontent.com/3102842/93359167-48a29080-f810-11ea-8691-569d0be8d84c.png">

Reply:
Neat, TIL! Thanks!